homeassistant/components/yamaha/media_player.py

```diff
@@ -138,21 +143,24 @@ async def async_setup_platform(
     # Keep track of configured receivers so that we don't end up
     # discovering a receiver dynamically that we have static config
     # for. Map each device from its zone_id .
-    known_zones = hass.data.setdefault(DATA_YAMAHA, set())
+    known_zones = hass.data.setdefault(DOMAIN, {KNOWN_ZONES: set()})[KNOWN_ZONES]
+    _LOGGER.debug("Known receiver zones: %s", known_zones)
 
     # Get the Infos for configuration from config (YAML) or Discovery
     config_info = YamahaConfigInfo(config=config, discovery_info=discovery_info)
     # Async check if the Receivers are there in the network
-    receivers = await hass.async_add_executor_job(_discovery, config_info)
+    zone_ctrls = await hass.async_add_executor_job(_discovery, config_info)
 
     entities = []
-    for receiver in receivers:
-        if config_info.zone_ignore and receiver.zone in config_info.zone_ignore:
+    for zctrl in zone_ctrls:
+        _LOGGER.info("Receiver zone: %s", zctrl.zone)
```

Comment:
Please don't log on info

Reply:
What are the acceptable uses of info logging

Reply:
but not here anyway